test/unit/person-test.js

```diff
@@ -38,85 +37,75 @@ describe('Person', function () {
   }
 
   describe('Searching by id', () => {
-    it('should work with a regid', async function () {
-      return await searchById({regid: '9136CCB8F66711D5BE060004AC494FFE'});
-    });
+    it('should work with a regid', async () => searchById({ regid: '9136CCB8F66711D5BE060004AC494FFE' }));
 
-    it('should work with a netid', async function () {
-      return await searchById({netid: 'javerage'});
-    });
+    it('should work with a netid', async () => searchById({ netid: 'javerage' }));
 
-    it('should work with an employee id', async function () {
-      return await searchById({employeeid: '000210620'});
-    });
+    it('should work with an employee id', async () => searchById({ employeeid: '000210620' }));
 
-    it('should work with student number', async function () {
-      return await searchById({studentNumber: '1313020'});
-    });
+    it('should work with student number', async () => searchById({ studentNumber: '1313020' }));
 
-    it('should work with system key', async function () {
-      return await searchById({syskey: '990003020'});
-    });
+    it('should work with system key', async () => searchById({ syskey: '990003020' }));
 
-    it('should work with development id', async function () {
-      return await searchById({devid: '9999999001'});
-    });
+    // it('should work with development id', async function () {
```

Comment:
do we wanna remove the commented out code?

Reply:
No. I need to get a new development id to re-enable the code. The old one is no longer valid for some reason.

Reply:
I went back and looked at the response and it looks like there may have been a change to the permissions for this data. I'm going to remove it after all.